src/providers/reddit_provider.ts

```diff
@@ -34,7 +36,7 @@ export default class RedditProvider extends Provider {
     });
 
     // Combine the user notifications
-    let notifications = [].concat(...userNotifications);
+    let notifications = merge(userNotifications);
```

Comment:
I like `merge()` but it needs a more descriptive name imo

Reply:
Fixed